doc/source/notebooks/advanced/mcmc.pct.py

```diff
@@ -624,25 +627,26 @@ def run_chain_fn():
 model = gpflow.models.GPR(data, kernel, meanf)
 model.likelihood.variance.assign(0.01)
 
-model.kernel.lengthscales.prior = tfd.Gamma(f64(1.0), f64(1.0))
-model.kernel.variance.prior = tfd.Gamma(f64(1.0), f64(1.0))
-model.likelihood.variance.prior = tfd.Gamma(f64(1.0), f64(1.0))
-model.mean_function.A.prior = tfd.Normal(f64(0.0), f64(10.0))
-model.mean_function.b.prior = tfd.Normal(f64(0.0), f64(10.0))
+mu = f64(0.0)
+std = f64(10.0)
+one = f64(1.0)
 
-# model.kernel.lengthscales.prior_on = "unconstrained"
-# model.kernel.lengthscales.prior = tfd.LogNormal(f64(0.0), f64(np.log(10.0)))
+model.kernel.lengthscales.prior_on = "unconstrained"
+model.kernel.lengthscales.prior = tfd.Normal(mu, std)
```

Comment:
To save instantiating tfd.Normal many times perhaps we can just call it once (or will this have some undesirable properties?)
```suggestion
default_prior = tfd.Normal(mu, std)
model.kernel.lengthscales.prior = default_prior
```